packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/use-entity-type-value.tsx

```diff
@@ -19,10 +25,47 @@ import {
   useFetchEntityTypes,
 } from "../../../../../shared/entity-types-context/hooks";
 
+interface EntityTypeAndPropertyTypes {
+  entityType: EntityType;
+  propertyTypes: Record<string, PropertyType>;
+}
+
+const getPropertTypes = (
+  properties: any,
+  subgraph: Subgraph,
+  propertyTypes?: Map<string, PropertyType>,
+) => {
+  let propertyTypesMap = propertyTypes ?? new Map<string, PropertyType>();
```

Comment:
`getPropertyTypesReferencedByEntityType` in the stdlib might help simplify this – it will return an array of property type ids referenced by an entity type, given a subgraph and an entity type id. 

This will include property types referenced by property types, which you might not need for this purpose, but probably don't hurt to include.

Reply:
It returns only the `baseId` and `version` of the property type in the entity type, and also does not return property types for `depth > 0`.